forte/data/data_store.py

```diff
@@ -937,7 +940,46 @@ def _parse_onto_file(self):
         """
         if self._onto_file_path is None:
             return
-        raise NotImplementedError
+
+        merged_scheme, merged_dict = [], []
+        DataStore.onto_gen.parse_ontology_spec(
+            self._onto_file_path, merged_scheme, merged_dict
+        )
+
+        for onto in merged_scheme:
+            entry_name = onto["entry_name"]
+            if entry_name in DataStore._type_attributes:
+                continue
+
+            attr_dict = {}
+            idx = constants.ATTR_BEGIN_INDEX
+            if "attributes" in onto:
+                for d in onto["attributes"]:
+                    name = d["name"]
+                    attr_dict[name] = idx
+                    idx += 1
```

Comment:
You have to pay attention to the details, what is the order of the attributes? Can this be consistent? If we rely on this `dict` order, then it is arbitrary and there is no consistency guarantee.